doc/dftb+/manual/dftbp.tex

```diff
@@ -3117,6 +3121,47 @@ \subsubsection{Generalized Born Model}
 }
 \end{verbatim}
 
+
+\subsubsection{Charge Model 5}
+\label{sec:dftbp.CM5}
+
+The charge model 5 (CM5)\cite{marenich2012} can be used to correct partial charges by
+%
+\begin{equation}
+   q^\text{CM5}_\text{A} = q_\text{A}
+   + \sum_{B} D_\text{A--B} \exp[-\alpha(R_\text{AB} - R^\text{cov}_\text{A} - R^\text{cov}_\text{B}]
+\end{equation}
+
+The pairwise parameters $D_\text{A--B}$ are fixed to the original published ones,
+while the exponent $\alpha$ and the covalent radii $R^\text{cov}_\text{A}$ default
+to the published parameters but can be adjusted in the input.
+
+\begin{ptable}
+  \kw{alpha} & r & & 1.30918451402600 & \\
+  \kw{Radii} & m & & \is{atomicRad\cb} & \\
+  \kw{Cutoff} & r & & 30.0 & \\
+\end{ptable}
+
+\begin{description}
+
+  \item[\is{alpha}] Exponent of the CM5 correction in inverse Bohr
```

Comment:
We should discuss how to handle inverse units in input

Reply:
We can either read it as one over alpha and make it length units or add new inverse length units for this purpose. Also you usually don't want to modify this parameter.

Reply:
I going with new units for inverse lengths.